Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
-import './Card.css';
+import './Card.scss';
+
+const CardContainer = props => {
+    return (
+        <div className = 'card-container' onClick ={clickAction}>
+            {props.children}
```

Comment:
Still can't believe you figured out props.children. I didn't pick this up till react was over.